src/client/Microsoft.Identity.Client/AuthScheme/TokenType.cs

```diff
@@ -32,6 +32,12 @@ internal enum TokenType
         /// Extension token type.
         /// Extension = 5
         /// </summary>
-        Extension = 5
+        Extension = 5,
```

Comment:
unrelated - @trwalke, what will happen if we create another extension ?

Reply:
@trwalke ^^